wgpu/tests/clear_texture.rs

```diff
@@ -308,7 +309,8 @@ fn clear_texture_tests(
 #[test]
 fn clear_texture_2d_uncompressed() {
     initialize_test(
-        TestParameters::default().features(wgpu::Features::CLEAR_TEXTURE),
+        TestParameters::default()
+            .features(wgpu::Features::CLEAR_TEXTURE | wgpu::Features::DEPTH32FLOAT_STENCIL8),
```

Comment:
This needs to separately ask for depth32float as an optional feature and then only test it if it is available. As it stands, we can't run this test on any platform without depth32float.

Reply:
Sorry, I don't know how to set optional feature in the tests module, is like this enough?
```rust
#[test]
fn clear_texture_d32_s8() {
    initialize_test(
        TestParameters::default()
        .features(wgpu::Features::CLEAR_TEXTURE | wgpu::Features::DEPTH32FLOAT_STENCIL8)
        .specific_failure(Some(wgpu::Backends::GL), None, None, false),
        |ctx| {
            clear_texture_tests(
		&ctx, &[wgpu::TextureFormat::Depth32FloatStencil8],
		false, false,
            );
        },
    )
}
```

Reply:
Hmm I forgot that the test doesn't have the ability to have optional features. That works fine, though you don't need the fail on GL there.